.github/workflows/ci.yaml

```diff
@@ -3,16 +3,17 @@ name: build
 on: push
```

Comment:
Hmm, unrelated to your change, but I guess we should change this to `on: [push, pull_request]` so that it triggers for PRs made from forks like this one was?